spec/controllers/queue_items_controller_spec.rb

```diff
@@ -105,6 +172,24 @@
         expect(response).to redirect_to login_path 
       end
     end
+
+    describe "POST update" do
+      it "should redirect to login page" do
+        post :update,user_id: user1, queue_items: [{id: 1, position: 3}, {id: 2, position: 1}, {id: 3, position: 2}]
+        expect(response).to redirect_to login_path 
+      end
+    end
+  end
+
+  context "queue_items that do not belong to user" do
+    describe "POST update" do
+      it "should not update where queue_items does not belong to signed-in user" do
```

Comment:
The word `should` should be deleted not added. Its not needed - read more about this http://betterspecs.org/#should